pkg/crc/constants/constants.go

```diff
@@ -200,5 +200,7 @@ func GetCRCWindowsTrayDownloadURL() string {
 }
 
 func GetMsiInstallPath() string {
-	return filepath.Join(os.Getenv("PROGRAMFILES"), "CodeReady Containers")
+	// In case of error path will be empty string and upperr layer will handle
+	currentExecutablePath, _ := os.Executable()
```

Comment:
Maybe log the error ? Otherwise lgtm

Reply:
should we do logging in the constants package? there's an import cycle if i try to import the logging package :(

Reply:
Updated, logging the error now and fixed the import cycle in the next commit.